src/main/java/com/mraof/minestuck/item/block/HorseClockItem.java

```diff
@@ -0,0 +1,33 @@
+package com.mraof.minestuck.item.block;
+
+import com.mraof.minestuck.block.HorseClockMultiblock;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.context.BlockPlaceContext;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class HorseClockItem extends MultiblockItem
+{
+	public HorseClockItem(HorseClockMultiblock multiblock, Properties properties)
+	{
+		super(multiblock, properties);
+	}
+	
+	@Override
+	protected boolean placeBlock(BlockPlaceContext context, BlockState newState)
+	{
+		boolean superBool = super.placeBlock(context, newState);
+		
+		if(superBool)
+		{
+			Player player = context.getPlayer();
+			ItemStack itemstack = context.getItemInHand();
+			
+			if (player == null || !player.getAbilities().instabuild) {
+				itemstack.shrink(1);
+			}
+		}
+		
+		return superBool;
+	}
```

Comment:
I can't tell why this is done. Should it not already be the case that one item is consumed if it is placed in survival?

Reply:
It extends MultiblockItem, which it itself does not have a super call. Meaning stack shrinking needs to be added back in

Reply:
Ah, that is a bug in the implementation of the multiblock item then. I'll push a fix on the main branch.

Reply:
Updated and now have deleted this class